README.md

```diff
@@ -48,11 +47,11 @@ sequenceDiagram
 ## Features
 
 Currently, DAL supports CRUD operations on persisted objects, where data is
-persisted in Postgres database or in-memory cache, depending on library user's
+persisted in Postgres database, depending on library user's
 choice.
```

Comment:
Since we do not currently support any database besides Postgres, I propose that **library user's choice** is not mentioned, because the library user won't have any choice